ui/guest/src/components/GuestProxy.js

```diff
@@ -126,29 +155,7 @@ export function GuestProxy(props) {
           // Update attribute(s)
           // $el.attr('data-craftercms-index', newIndex);
 
-          forEach(
-            $el.parent().children(),
-            (el, i) => {
-
-              $(el).attr('data-craftercms-index', i);
-
-              const pr = ElementRegistry.fromElement(el);
-
-              context.deregister(pr.id);
-              registerElement(el);
-
-              if (
-                (newIndex > index && i >= newIndex) ||
-                (newIndex < index && i >= index)
-              ) {
-                console.log(
-                  `(newIndex > index && i >= newIndex) ${(newIndex > index && i >= newIndex)}`,
-                  `(newIndex < index && i >= index) ${(newIndex < index && i >= index)}`
-                );
-                return 'break';
-              }
-
-            });
+          registerDeregisterElements(Array.from($el.parent().children()), 'move', newIndex, index);
```

Comment:
I think jQuery does give you an array, no?

Reply:
no, it returns a htmlCollection